README.md

```diff
@@ -87,6 +90,7 @@ fn main() {
 ---
 
 ### ❖ What's New? 
+0.5.1 - Added Gruvbox Light, Biscuit Dark, Biscuit Light 
```

Comment:
this would likely be `0.6.0` since we're adding new kolorschemes and "removing" the old `gruvbox`

The line below can be removed as well

Reply:
The version should also be updated in the `Installation` part of the readme, as well as `cargo.toml`